ci/travis/osx/before_install.sh

```diff
@@ -28,6 +28,7 @@ brew ln expat --force
 brew ln libxml2 --force
 brew ln gettext --force
 brew ln libffi --force
+brew remove gdal
 
 mkdir -p ${HOME}/Library/Python/2.7/lib/python/site-packages
 echo 'import site; site.addsitedir("/usr/local/lib/python2.7/site-packages")' >> ${HOME}/Library/Python/2.7/lib/python/site-packages/homebrew.pth
```

Comment:
`/usr/local/opt/gdal-20/lib/python2.7/site-packages` needs added to PYTHONPATH (or as part of a .pth file) here, since the current `gdal-20` formula is _keg-only_, i.e. it's packages are not symlinked into `/usr/local/lib/python2.7/site-packages`.